unit_test/entities/test_pipe.py

```diff
@@ -21,38 +21,83 @@
 from omotes_simulator_core.entities.assets.pipe import Pipe
 from omotes_simulator_core.entities.esdl_object import EsdlObject
 from omotes_simulator_core.infrastructure.utils import pyesdl_from_file
+from omotes_simulator_core.solver.network.assets.solver_pipe import SolverPipe
 
 
 class PipeTest(unittest.TestCase):
     """Testcase for HeatNetwork class."""
 
+    def setUp(self):
+        """Define the variables used in the tests."""
+        self.length: float = 1
+        self.inner_diameter: float = 1
+        self.roughness: float = 0.001
+        self.alpha_value: float = 0.8
+        self.minor_loss_coefficient: float = 0.0
+        self.external_temperature: float = 273.15 + 20.0
+        self.qheat_external: float = 0.0
+
     def test_pipe_create(self):
         """Evaluate the creation of a pipe object."""
-        # Arrange
-
         # Act
-        pipe = Pipe(asset_name="pipe", asset_id="pipe_id", port_ids=["test1", "test2"])
-
+        pipe = Pipe(
+            asset_name="pipe",
+            asset_id="pipe_id",
+            port_ids=["test1", "test2"],
+            length=self.length,
+            inner_diameter=self.inner_diameter,
+            roughness=self.roughness,
+            alpha_value=self.alpha_value,
+            minor_loss_coefficient=self.minor_loss_coefficient,
+            external_temperature=self.external_temperature,
+            qheat_external=self.qheat_external,
+        )
+        solver_pipe = SolverPipe(name="pipe", _id="pipe_id", length=pipe.length)
```

Comment:
Why did you add this? Now you have to acts in your act block. You should only have one act statement. The solver pipe is tested in its own test case

Reply:
completed.